lib/CLIManager.js

```diff
@@ -369,25 +384,47 @@ class CLIManager {
 
   /**
    * Handles any result from command execution.
-   * @param err
-   * @param result
+   * @param {CommandResult} result
    * @param [done]
    */
-  processCommandResult(err, result, done) {
+  processCommandResult(result, done) {
     this._clearSession(() => {
       const isCallbackProvided = typeof done === 'function';
-      if (err) {
-        this.log(LogLevel.ERROR, '%s', err);
+      if (isCallbackProvided) {
```

Comment:
It's a leftover from an older CLI version. Given our offline discussion regarding refactoring logic related to commands handling, do you think it will be OK to be fixed in the future? As this is code, that will be changed in the end.

Reply:
ok.